improver_tests/calibration/ensemble_calibration/test_ApplyEMOS.py

```diff
@@ -182,7 +158,8 @@ def test_bias(self):
 
     def test_spread(self):
         """Test emos coefficients that correct underspread"""
-        self.coefficients.data = [1, 1, 0, 1]
+        for coeff, cube in zip([1, 1, 0, 1], self.coefficients):
+            cube.data = coeff
```

Comment:
As above - you are only updating the one (gamma) coefficient here.

Reply:
Amended.